src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const averageAgeMan = people
+    .filter((person) => century ? person.sex === 'm'
```

Comment:
sex === 'm' and sex === 'm' are duplicated in other places, so please create separate functions 
`isMale` and `isFemale` and reuse them